libs/json/src/test/test_json_simple.c

```diff
@@ -205,4 +209,32 @@ TEST_CASE(test_json_simple_decode){
 
     rc = memcmp(string2, "foobarlongstring", 10);
     TEST_ASSERT(rc==0);
-}
+        
+        //second stuffs to attr
```

Comment:
Hi nges, according to Coding style, C++ style comments (//) are not permitted.  Please use C style comments /* */